_posts/2020-01-17-event-store-6.0.0-preview-2-release.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Event Store 6.0.0 Preview 2"
+author: "Mat McLoughlin"
+layout: blog-post
+category: "Release Notes"
+---
+
+We are excited to announce the second preview release of Event Store 6.0.0.
+
+With this release we want to show you where we are planning on taking Event Store in the future.
```

Comment:
I think we can remove this line. What do you think? We said something similar in the previous release.